anvil/src/main/kotlin/trikita/anvil/CustomDslSetter.kt

```diff
@@ -72,6 +74,7 @@ fun ViewScope.toStartOf(subject: Int) = align(RelativeLayout.START_OF, subject)
 fun ViewScope.align(verb: Int, subject: Int) = attr("align", verb to subject)
 
 fun ViewScope.anim(animator: Animator, trigger: Boolean) = attr("anim", AnimatorPair(animator, trigger))
+fun ViewScope.animBi(trigger: Boolean, start: Animator, finish: Animator) = attr("animBidirectional", AnimatorTriple(start, finish, trigger))
```

Comment:
Maybe the same name as the previous one, `anim`?
It also seems logical to make `trigger` the first parameter -- this way both methods would be accessible via contextual suggestions similarly.

Reply:
Maybe even same attribute name, we still will be able to differentiate by argument type.